Docker/README.md

```diff
@@ -0,0 +1,278 @@
+    * container: running instance of an image
+
+
+<a name="readme-template">
+
+## Proposed README.md template for DI Docker images ##
+
+```
+# Title describing what the DI Docker image is/does/contains #
+
+
+## What is this? ##
+
+A paragraph or two describing the purpose/goal of this Docker project. What need does it fulfill?  
+
+
+## Requirements ##
+
+What is required to work with this Docker container? What are the prerequisites?
+
+
+## In this Docker Container ##
+
+What is in the Docker Container? Aim, give the reader a brief idea at a high level what the Dockerfile builds without the user having to read and understand the Dockerfile. Reasoning for use the given base image?
+
+
+## Usage ##
+
+Audience: individual with little Docker experience or limited Linux admin experience. Include any special instructions otherwise link to the general [Developer Handbook Docker Usage Section](https://github.com/ualbertalib/Developer-Handbook/blob/master/docker/README.md#Usage)
+
+### Container creation ###
+
+Steps to run the container. 
+
+### Other usage instructions, if applicable ###
+
+For example, how to run test tasks within the container or how to setup for debugging.
+
+
+## Maintenance ##
+
+Include maintenance tasks e.g., update Docker Hub
+
+### Updating Docker Hub ###
+
+* Associated Docker Hub repository URL
+* Any special instructions on updating Docker Hub otherwise link to the general [docker stop, rm, pull new version, run notes](#Upgrading-local-container)
+
+### Upgrading local container ###
+
+Any special instructions on updating a local container otherwise link to the general [docker stop, rm, pull new version, run notes](#Upgrading-local-container)
+
+
+## Frequently used commands ##
+
+A list of frequently used commands specific to this image. Link to [Developer Handbook](https://github.com/ualbertalib/Developer-Handbook/blob/master/docker/README.md#Frequently-used-commands)
+
+## Special notes / warnings / gotchas ##
+
+Notes to leave to a future user/maintainer that might hightlights something counter-intuitive.
+
+
+## Future considerations ##
+
+To capture any proposed future work or future considerations.
+
+```
+
+
+
+## Notes around practice ## 
+
+* Take care around the use the Docker Hub ":latest" tag because it is "the last build/tag that ran without a specific tag/version specified" [reference](https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375)
+
+* link/depends_on in docker-compose to handle dependencies and determine the order of service start-up.
+  * [links](https://docs.docker.com/compose/compose-file/compose-file-v2/#links)
+  * [depends_on](https://docs.docker.com/compose/compose-file/compose-file-v2/#dependson)
+
+
+## Special notes / warnings / gotchas ##
+
+
+
+## Future considerations ##
+
+* Consider [configuration of automated builds on Docker Hub](https://docs.docker.com/docker-hub/builds/)
+
```

Comment:
Maybe add Docker Swarm/Atomic as mentioned here: https://github.com/ualbertalib/di_docker_images/issues/10 and https://github.com/ualbertalib/di_docker_images/issues/11 as future consideration